src/main/java/gov/usgs/owi/nldi/controllers/RedirectController.java

```diff
@@ -0,0 +1,30 @@
+package gov.usgs.owi.nldi.controllers;
+
+import gov.usgs.owi.nldi.services.ConfigurationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.view.RedirectView;
+
+import io.swagger.v3.oas.annotations.Hidden;
+
+@RestController
+public class RedirectController {
+
+	protected final ConfigurationService configurationService;
+
+	@Autowired
+	public RedirectController(ConfigurationService configurationService) {
+		this.configurationService = configurationService;
+	}
+
+
+	@GetMapping(value="/swagger")
+	@Hidden
+	public RedirectView getSwagger() {
+		return new RedirectView(configurationService.getRootUrl() + "/swagger-ui/index.html?configUrl="
+									+ configurationService.getSwaggerApiDocsUrl(), true, true);
+	}
+
+
+}
```

Comment:
I like blank lines at the end of the file. GitHub has also flagged this.

Reply:
Yeah, I think the IntelliJ defaults are not helping me.  I'll see if I can get blank lines added automatically on save.